### PR TITLE
LRDOCS 8105 & LRDOCS 8449 Activating Commerce - SME

### PR DESCRIPTION
**Pull request by JamesAGarcia**
[LRDOCS-8105](https://issues.liferay.com/browse/LRDOCS-8105)
[LRDOCS-8449](https://issues.liferay.com/browse/LRDOCS-8449)
This PR edits both _Installation Overview_  and _Activating Liferay Commerce Enterprise_ articles per SME feedback while condensing the _Updating a Commerce Enterprise License_ article back into _Activating Liferay Commerce Enterprise_.

### CODE REVIEW

File: docs/commerce/2.x/en/installation-and-upgrades/activating-liferay-commerce-enterprise.md [hunk 1]
```diff
@@ -2,62 +2,55 @@
...
+Liferay Commerce Enterprise is built on Liferay DXP and requires active DXP and Commerce licenses for use. These licenses are provided as XML (`.xml`) activation keys that must be deployed to your DXP instance to activate all Commerce features.
 
-* Open a [Help Center](https://liferay-support.zendesk.com/agent/) ticket with the *Activation Key/Project Administration* component.
-
-* Send a request by email to your regional Provisioning team (i.e., provisioning-[region]@liferay.com).
-
-* Download a Commerce activation key from [here](https://customer.liferay.com/en_US/activation-key).
+Commerce licenses use many of the same parameters as DXP licenses, including `product-version`, `license-type`, and `expiration-date`. However, restrictions based on System resources (e.g., processor cores) or product version are not implemented in Commerce licenses.
```
Comment (jrhoun) [inline docs/commerce/2.x/en/installation-and-upgrades/activating-liferay-commerce-enterprise.md:7]: > restrictions based on System resources

It doesn't make sense to me to capitalize "System" here. Is there something I'm missing?

File: docs/commerce/2.x/en/installation-and-upgrades/activating-liferay-commerce-enterprise.md [hunk 2]
```diff
@@ -66,33 +59,38 @@ Follow these steps to deploy the key to a Docker container using the `docker cp`
    INFO  [fileinstall-directory-watcher][LicenseManager:?] License registered for Commerce Subscription Development
    ```
 
-## Deploying to DXP Cloud
+## Updating You Commerce Enterprise License
```
Comment (jrhoun) [inline docs/commerce/2.x/en/installation-and-upgrades/activating-liferay-commerce-enterprise.md:62]: > ## Updating You Commerce Enterprise License

Typo here -- but also I'd rather it be more specific of a header: "Updating an Expired Commerce Enterprise License" perhaps?

File: docs/commerce/2.x/en/installation-and-upgrades/activating-liferay-commerce-enterprise.md (hunk 1 above)
Comment (JamesAGarcia) [reply docs/commerce/2.x/en/installation-and-upgrades/activating-liferay-commerce-enterprise.md:7]: I can't remember where I got that statement from, but it doesn't need to be capitalized.

File: docs/commerce/2.x/en/installation-and-upgrades/activating-liferay-commerce-enterprise.md (hunk 2 above)
Comment (JamesAGarcia) [reply docs/commerce/2.x/en/installation-and-upgrades/activating-liferay-commerce-enterprise.md:62]: Agreed

File: docs/commerce/2.x/en/installation-and-upgrades/activating-liferay-commerce-enterprise.md [hunk 3]
```diff
@@ -2,62 +2,55 @@
...
 
-Commerce licenses use many of the same parameters as DXP licenses, including `product-version`, `license-type`, and `expiration-date`. Restrictions based on System resources (e.g., processor cores) or product version are not implemented in Commerce licenses.
+Both the DXP and Commerce activation keys must be of the same `license-type` (e.g., `production`, `developer`, or `enterprise`). A warning is thrown in the server startup log if the license types do not match. *Production* licenses also require a matching hostname, matching IP address or matching mac address for validation.
 
-```important::
-   In order to activate Commerce Enterprise, both the DXP and Commerce activation keys must be of the same `license-type` (i.e., Production, Developer, or Enterprise). A warning is thrown in the server startup log if the license types do not match.
+```note::
+   As of Liferay 7.3 SP1, only LCS can be used to activate Commerce `enterprise` or `production` licenses. The same is not true of `developer` licenses.
```
[the remark this answers is not in the document]
Comment (JamesAGarcia) [reply docs/commerce/2.x/en/installation-and-upgrades/activating-liferay-commerce-enterprise.md:12]: The SME stated: 
> LCS alone can be used to activate Commerce enterprise or production licenses (but not commerce developer)
I can follow up with him to clarify this point.

[the remark this answers is not in the document]
Comment (JamesAGarcia) [reply docs/commerce/2.x/en/installation-and-upgrades/activating-liferay-commerce-enterprise.md:12]: @jrhoun Just talked with the SME, and this text is misleading/inaccurate. I'll update the PR with a revised admonition (along with your other requested edits).

**Comment by JamesAGarcia** [comment]
I updated the PR with the requested edits.

**Comment by jrhoun** [comment]
Merged. Thank you.

**Pull request closed without merging**